api/src/main/java/org/pmiops/workbench/db/model/DbAccessModule.java

```diff
@@ -45,7 +45,7 @@ public DbAccessModule setExpirable(boolean expirable) {
 
   @Column(name = "bypassable", nullable = false)
   public boolean getBypassable() {
-    return bypassable;
+    return true;
```

Comment:
This is the core change. All modules are now bypassable, but the DB doesn't reflect this yet. In follow up PRs I plan to remove this column entirely.